ztp/ztp-ran-policy-generator/README.md

```diff
@@ -1,11 +1,265 @@
+                    exclude:
+                        - kube-*
+                    include:
+                        - '*'
+                object-templates:
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: ptp.openshift.io/v1
+                        kind: PtpConfig
+                        metadata:
+                            name: slave
+                            namespace: openshift-ptp
+                        spec:
+                            recommend:
+                                - match:
+                                    - nodeLabel: node-role.kubernetes.io/worker-du
+                                  priority: 4
+                                  profile: slave
+                            profile:
+                                - interface: ens5f0
+                                  name: slave
+                                  phc2sysOpts: -a -r -n 24
+                                  ptp4lOpts: -2 -s --summary_interval -4
+                                  ptp4lConf: |
+                                    [global]
+                                    #
+                                    # Default Data Set
+                                    #
+                                    twoStepFlag 1
+                                    slaveOnly 0
+                                    priority1 128
+                                    .....
+```
+
+# Install and execute
 -  We assume kustomize and golang are installed
--  to build the plugin
+-  Build the plugin
 
     - $ cd ztp/ztp-ran-policy-generator/kustomize/plugin/ranPolicyGenerator/v1/ranpolicygenerator/
     - $ go build -o RanPolicyGenerator
 
--  to execute kustomize
+
+- The [kustomization.yaml](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/ztp-ran-policy-generator/kustomization.yaml) has a reference to the ranPolicyGenerator.yaml . The RanPolicyGenerator definition as below
+
+```
+apiVersion: ranPolicyGenerator/v1
+kind: RanPolicyGenerator
+metadata:
+  name: acm-ran-policy
+  namespace: ran-policy-generator
+# The arguments should be given and defined as below with same order --ranGenPath= --sourcePath= --outPath= --stdout --customResources
+argsOneLiner: ./ranGenTemplateExamples ./sourcePolicies ./out true false
+```
+- The RanPolicyGenerator parameters are defined as below:
+
+    - ranGenPath: path to the RanGenTemp files
+    - sourcePath: path to the source policies
+    - outPath: path to save the generated ACM policies 
+    - stdout: if true will print the generated policies to console
+    - customResourceOnly: if true will generate the CRs from the source Policies files only without ACM policies. 
+
+- Test RanGen by executing the below commands:
 
     - $ cd cnf-features-deploy/ztp/ztp-ran-policy-generator/
     - $ XDG_CONFIG_HOME=./ kustomize build --enable-alpha-plugins
+
+    You should have out directory created with the expected policies
```

Comment:
Maybe mention the structure that the output directory will have?

Reply:
done, add the expected output tree